pool/getter.gno

```diff
@@ -14,6 +15,42 @@ func PoolGetPoolList() []string {
 	return poolPaths
 }
 
+func PoolGetSlot0Tick(poolPath string) int32 {
+	pool, exist := pools[poolPath]
+	if !exist {
+		panic(addDetailToError(
+			errDataNotFound,
+			ufmt.Sprintf("_GET_no_receiver.gno()__PoolGetSlot0Tick || poolPath(%s) does not exist", poolPath),
+		))
+	}
+
+	return pool.slot0.tick
+}
+
+func PoolGetSqrtPriceX96(poolPath string) *u256.Uint {
+	pool, exist := pools[poolPath]
+	if !exist {
+		panic(addDetailToError(
+			errDataNotFound,
+			ufmt.Sprintf("_GET_no_receiver.gno()__PoolGetSqrtPriceX96 || poolPath(%s) does not exist", poolPath),
+		))
+	}
+
+	return pool.slot0.sqrtPriceX96
+}
+
+func PoolGetLiquidity(poolPath string) string {
+	pool, exist := pools[poolPath]
+	if !exist {
+		panic(addDetailToError(
+			errDataNotFound,
+			ufmt.Sprintf("_GET_no_receiver.gno()__PoolGetLiquidity || poolPath(%s) does not exist", poolPath),
+		))
+	}
+
+	return pool.liquidity.ToString()
+}
+
 func (pool *Pool) PoolGetSlot0() Slot0 {
```

Comment:
It's probably a good idea not to prefix a receiver function with the same name as the object.
Since this is a receiver function for a Pool object, we need to remove Pool as the function's prefix.

Reply:
If this comment has been resolved, it would be okay to merge

Reply:
d1fc3b1f1834e432e2f0294e752b6e3d0900adf3